addon/components/sl-radio-group.js

```diff
@@ -56,6 +56,45 @@ export default Ember.Component.extend( InputBased, TooltipEnabled, {
     inline: null,
 
     /**
+     * The component's text label
+     *
+     * @type {?String}
+     */
+    label: null,
+
+    /**
+     * The "name" attribute for the children radio buttons
+     *
+     * Similar to the `inline` property; the default value is null, so the
+     * children buttons will not be overridden. Any string value will overwrite
+     * the children buttons' name property.
+     *
+     * @type {?String}
+     */
+    name: null,
+
+    /**
+     * To style the component to reflect that it has optional status.
+     *
+     * @type {Boolean}
+     */
+    optional: false,
+
+    /**
+     * Whether the radio buttons should be readonly
+     *
+     * @type {Boolean}
+     */
+    readonly: false,
```

Comment:
readonly HTML5This Boolean attribute indicates that the user cannot modify the value of the control. It is ignored if the value of the type attribute is hidden, range, color, checkbox, radio, file, or a button type (such as button or submit).